CHANGES.md

```diff
@@ -18,12 +18,21 @@ To be released.
 
 ### Behavioral changes
 
+ -  Inner logic of `ByteUtil.CalculateHashCode(byte[] bytes)` has modified.
+    [[#1866], [#1891]]
+
 ### Bug fixes
 
 ### Dependencies
 
+ -  No longer depend on *Fody*.  [[#1866], [#1891]]
+ -  No longer depend on *Fody.Equals*.  [[#1866], [#1891]]
```

Comment:
It's confusing, but the package is named the other way.  See also: <https://www.nuget.org/packages/Equals.Fody>.

```suggestion
 -  No longer depend on *Equals.Fody*.  [[#1866], [#1891]]
```

Reply:
Ooops my bad 😥

Reply:
It's not your bad, but their bad!  😇